src/components/NucleotideEntropyMultiChart.tsx

```diff
@@ -0,0 +1,294 @@
+    };
+
+    const exploreUrl = useExploreUrl()!;
+    const variants = exploreUrl.variants?.map(nonNullValues);
+
+    let selectedGene: GeneOption[] = genes.filter(g => g.name === gene).map(g => {
+        return {
+            value: g.name,
+            label: g.name,
+            startPosition: g.startPosition,
+            endPosition: g.endPosition,
+            aaSeq: g.aaSeq,
+            color: ''
+          };
+    })
+
+    const data = useData(selectors, selectedGene, variants, sequenceType, deletions);
+
+    const controls = (
+        <div className='mb-4'>
+          {/* AA vs. nucs */}
+          <div className='mb-2 flex'>
+            <PipeDividedOptionsButtons
+              options={[
+                { label: 'NUC', value: 'nuc' },
+                { label: 'AA', value: 'aa' },
+              ]}
+              selected={sequenceType}
+              onSelect={setSequenceType}
+            />
+          </div>
+          {/* Include deletions */}
+            <FormGroup>
+              <FormControlLabel
+                control={
+                  <Checkbox
+                    checked={deletions}
+                    onChange={handleDeletions}
+                    inputProps={{ 'aria-label': 'controlled' }}
+                  />
+                }
+                label='Include deletions'
+              />
+            </FormGroup>
+          {/* Genes */}
+          {
+            <div className=' w-72 flex mb-2'>
+              <div className='mr-2'>Gene:</div>
+              <Form.Control
+                as='select'
+                value={gene}
+                onChange={ev => setGene(ev.target.value)}
+                className='flex-grow'
+                size='sm'
+              >
+                {jsonRefData.genes.map(g => (
+                  <option value={g.name} key={g?.name}>
+                    {g.name}
+                  </option>
+                ))}
+              </Form.Control>
+            </div>
+          }
+        </div>
+    );
+
+
+
+    let plotArea;
+    if (variants![0].length == 0) {
+      plotArea = <p>Select one or more lineages to compare.</p>
+    } else if (!data) {
+      plotArea = <Loader />;
+    } else {
+        plotArea = (
+        <Plot
+            plotData={data.dayArray}
+            ticks={data.ticks}
+            variants={variants!}
+            sequenceType={sequenceType}
+        />
+        );
+    }
+    
+    return (
+        <>
+        {controls}
+        {plotArea}
+        </>
+      );
+}
+
+type PlotProps = {
+    plotData: TransformedTime;
+    ticks: string[];
+    variants: string[];
+    sequenceType: SequenceType;
+};
+
+const useData = (
+    selectors: LapisSelector[],
+    selectedGene: GeneOption[],
+    variants: any,
+    sequenceType: SequenceType,
+    deletions: boolean
+): any | undefined => {
+  //fetch the proportions per position in weekly segments
+  const weeklyVariantMutationProportionQuery = useQuery(
```

Comment:
Let's also get rid of the comments here: :) 
```suggestion
  const weeklyVariantMutationProportions = useProportionsPerPositionInWeeklySegments(selectors, sequenceType);

```